### PR TITLE
added correct redis command in compose to read conf file on startup

### LINKED ISSUE #558: Small issue with redis not reading the configuration file correctly
**Describe the bug**

**Affected Component(s)**
- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

**To Reproduce**

**Expected behavior**

**Screenshots**

**System (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by vkotronis**
### Description of PR

What component(s) does this PR affect?

- [X] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #558

### Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [X] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docker-compose.benchmark.yaml [hunk 1]
```diff
@@ -387,6 +387,7 @@ services:
             - ${REDIS_PORT}
         volumes:
             - ./backend-services/redis/configs/redis.conf:/usr/local/etc/redis/redis.conf
+        command: ["redis-server", "/usr/local/etc/redis/redis.conf"]
```
Comment (vkotronis) [inline docker-compose.benchmark.yaml:390]: @slowr could you add this command in `redis-fix` branch to the proper k8s container place and test? It should have the same effect as in compose. Thanks!

File: docker-compose.yaml [hunk 2]
```diff
@@ -372,6 +372,7 @@ services:
             - ${REDIS_PORT}
         volumes:
             - ./local_configs/backend/redis.conf:/usr/local/etc/redis/redis.conf
+        command: ["redis-server", "/usr/local/etc/redis/redis.conf"]
```
Comment (vkotronis) [inline docker-compose.yaml:375]: @slowr could you add this command in redis-fix branch to the proper k8s container place and test? It should have the same effect as in compose. Thanks!

**Comment by vkotronis** [comment]
@slowr do you agree with this PR? Any suggestions/comments? After I merge it we can check the other one that uses redis key expiry events to restart taps that do not receive any data for certain time periods.

**Review by slowr — APPROVED** [review]
LGTM!

**Pull request merged**